webui/src/pages/extension-detail/extension-detail.tsx

```diff
@@ -211,64 +215,66 @@ export class ExtensionDetailComponent extends React.Component<ExtensionDetailCom
+                    <Typography classes={{ root: this.props.classes.description }}>{extension.description}</Typography>
                 </Box>
-                <TextDivider theme={extension.galleryTheme} />
-                <Box className={this.props.classes.alignVertically}>{this.renderLicense(extension, themeClass)}</Box>
-            </Box>
-            <Box mt={2} mb={2} overflow='auto'>
-                <Typography classes={{ root: this.props.classes.description }}>{extension.description}</Typography>
-            </Box>
-            <Box display='flex' className={`${themeClass} ${this.props.classes.count}`}>
-                <Box className={this.props.classes.alignVertically}>
-                    <SaveAltIcon fontSize='small' />&nbsp;{extension.downloadCount || 0}&nbsp;{extension.downloadCount === 1 ? 'download' : 'downloads'}
-                </Box>
-                <TextDivider theme={extension.galleryTheme} />
-                <RouteLink
-                    to={createRoute([ExtensionDetailRoutes.ROOT, extension.namespace, extension.name, ExtensionDetailRoutes.TAB_REVIEWS])}
-                    className={`${this.props.classes.link} ${themeClass}`}
-                    title={
-                        extension.averageRating !== undefined ?
-                            `Average rating: ${this.getRoundedRating(extension.averageRating)} out of 5`
-                            : 'Not rated yet'
-                    }>
+                <Box display='flex' className={`${themeClass} ${this.props.classes.count}`}>
                     <Box className={this.props.classes.alignVertically}>
-                        <ExportRatingStars number={extension.averageRating || 0} fontSize='small' />
-                        ({extension.reviewCount})
+                        <SaveAltIcon fontSize='small' />&nbsp;{extension.downloadCount || 0}&nbsp;{extension.downloadCount === 1 ? 'download' : 'downloads'}
                     </Box>
-                </RouteLink>
+                    <TextDivider theme={extension.galleryTheme} />
+                    <RouteLink
+                        to={createRoute([ExtensionDetailRoutes.ROOT, extension.namespace, extension.name, ExtensionDetailRoutes.TAB_REVIEWS])}
+                        className={`${this.props.classes.link} ${themeClass}`}
+                        title={
+                            extension.averageRating !== undefined ?
+                                `Average rating: ${this.getRoundedRating(extension.averageRating)} out of 5`
+                                : 'Not rated yet'
+                        }>
+                        <Box className={this.props.classes.alignVertically}>
+                            <ExportRatingStars number={extension.averageRating || 0} fontSize='small' />
+                            ({extension.reviewCount})
+                        </Box>
+                    </RouteLink>
+                </Box>
             </Box>
-        </Box>;
+        );
```

Comment:
Have you changed the formatting in this method, or why are so many lines marked as changed?

I changed some details on the rendering of this header area, please rebase your branch on the latest master.

Reply:
@spoenemann Thanks. Done.